contrib/logging/fluentd-gcp-image/Dockerfile

```diff
@@ -14,18 +14,9 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV OPTS_APT -y --force-yes --no-install-recommends
 
 RUN apt-get -q update && \
-    apt-get -y install apt-utils adduser && \
-    apt-get clean
-
-ADD google-fluentd_1.0.0-0_amd64.deb /etc/google-fluentd/pkg/google-fluentd_1.0.0-0_amd64.deb
-RUN dpkg -i /etc/google-fluentd/pkg/google-fluentd_1.0.0-0_amd64.deb
-RUN /opt/google-fluentd/embedded/bin/gem install google-api-client
-ADD out_google_cloud.rb /etc/google-fluentd/plugin/out_google_cloud.rb 
-ADD agent.conf /etc/google-fluentd/google-fluentd.conf
-COPY catch-all-inputs.tar.gz /tmp/catch-all-inputs.tar.gz 
-RUN tar -C /etc/google-fluentd -zxf /tmp/catch-all-inputs.tar.gz
-RUN sed -i~ -e "s/\(USER\|GROUP\)=google-fluentd/\1=root/;" /etc/init.d/google-fluentd
-RUN sed -i~ -e 's/ --use-v1-config//' /etc/init.d/google-fluentd
+    apt-get -y install curl && \
+    apt-get clean && \
+    curl -s https://storage.googleapis.com/signals-agents/logging/google-fluentd-install.sh | sudo bash
```

Comment:
Yes, the it does actually require bash now, it will not work with sh as you noted...  although that's only due to the use of [[ rather than [, which probably isn't strictly necessary.